### PR TITLE
More pcap_ to pcapint_ for non-API routines.

### PR DESCRIPTION
**Pull request by guyharris**

### CODE REVIEW

File: missing/strlcpy.c [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*	$OpenBSD: pcap_strlcpy.c,v 1.12 2015/01/15 03:54:12 millert Exp $	*/
+/*	$OpenBSD: strlcpy.c,v 1.12 2015/01/15 03:54:12 millert Exp $	*/
```
Comment (infrastation) [inline missing/strlcpy.c:1]: Search and replace misfired here (possibly somewhere else too).

Comment (guyharris) [reply missing/strlcpy.c:1]: Nope, that's a fix to an earlier search-and-replace from a while ago that shouldn't have happened in the first place; we (somebody, probably me) grabbed the OpenBSD strlcpy.c, renamed the routine, and added it.  I fixed the /* OpenBSD */ comment to use the original source file's name.

Comment (infrastation) [reply missing/strlcpy.c:1]: I stand corrected.

Comment (guyharris) [reply missing/strlcpy.c:1]: It was probably my screwup in the first place, so don't worry about it.

File: scanner.l [hunk 2]
```diff
@@ -658,6 +658,7 @@ stou(const char *yytext_arg, YYSTYPE *yylval_arg, compiler_state_t *yyextra_arg)
 {
 	stoulen_ret ret;
 
+fprintf(stderr, "%s\n", yytext_arg);
```
Comment (fxlb) [inline scanner.l:661]: Should be removed from this PR.

Comment (guyharris) [reply scanner.l]: Yeah, that was me trying to figure out what that memory sanitizer issue was about.  Removed.

Comment (fxlb) [reply scanner.l]: I had the same...

**Review by mcr — APPROVED** [review]

**Pull request merged**